docs/antrea-network-policy.md

```diff
@@ -774,6 +776,10 @@ is provided, the label will be added in the log. For example, in the
 [ACNP with log settings](#acnp-with-log-settings), traffic that hits the
 "AllowFromFrontend" rule will be logged with log label "frontend-allowed".
```

Comment:
Shall we also mention `best-effort` in this paragraph, around third line? To keep it updated and straightforward.

Reply:
done